ckanext/iati/logic/validators.py

```diff
@@ -33,9 +44,14 @@ def iati_resource_url(value, context):
 
     valid_schemes = ('http', 'https', 'ftp')
     if not url.scheme in valid_schemes:
+        send_url_invalid_email(context)
```

Comment:
Does it make sense to assemble all of the possible issues so that they can deal with it in one shot?

Reply:
What do you mean?
I guess on the next line it will rase and exception so dont we want to send an email before that?